src/components/ebay-radio/index.marko

```diff
@@ -11,7 +11,8 @@ $ input.toJSON = noop;
         type="radio"
         class="radio__control"
         onClick("handleClick")
-        onFocus("handleFocus")/>
+        onFocus("handleFocus")
+        onkeydown("handleKeyDown")/>
```

Comment:
Needs to be camel case. onKeydown

Reply:
So I would agree, however when I switch that to camelcase it doesn't seem to work. 

I add an example template.marko to check
```marko
class {
    handleKeyDown(e){console.log(e);}
}

<ebay-radio value="yes" name="radio" on-keydown('handleKeyDown')/>

```

and nothing prints when I use `onKeydown` in the index.marko, but `onkeydown` works.

Reply:
wait hold on

Reply:
you're right, changing...